ledger/caching/src/main/scala/com/daml/caching/Cache.scala

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
What was the reason for moving this file into a different module?

Reply:
I believe it's because nothing in _libs-scala_ depends on Dropwizard Metrics.

Reply:
Because I had to create a dependency with `//ledger/metrics` and `//libs-scala` should contain only libraries with no internal dependencies. But good point to raise, thanks.